gallery/visualtests/nxGlobalSidebar.js

```diff
@@ -48,5 +48,16 @@ describe('nx-global-sidebar', function() {
     it('has a blue background when selected', simpleTest(selectedLink));
   });
 
+  describe('with a different logo size', function() {
+    const { getPage, checkFullPageScreenshot } = setupBrowser('#/NxGlobalSidebarScrollingExample', false);
+
+    it('looks right', async function() {
+      await getPage().setViewport({ width: 1366, height: 1000 });
+      await checkFullPageScreenshot();
+    });
+
+    it('passes a11y checks', a11yTest(null, true));
```

Comment:
Probably not necessary

Reply:
>Probably not necessary

Removed [2ec3b0a](https://github.com/sonatype/sonatype-react-shared-components/pull/1078/commits/2ec3b0a1cc54504504e02aeb94151a7fa64c8755)